.github/workflows/build.yml

```diff
@@ -146,10 +146,18 @@ jobs:
         working-directory: strains/${{ inputs.STRAIN_PATH }}/${{ env.TUTOR_APP_NAME }}
         env:
           SERVICE: ${{ inputs.SERVICE }}
+          LOGS_FILE_PATH: /tmp/build_logs.out
         run: |
           . .tvm/bin/activate
           tutor config save
-          tutor images build $SERVICE --no-cache
+          (tutor images build $SERVICE --no-cache 2>&1) | tee $LOGS_FILE_PATH
+      
+      - name: Identify silent errors
```

Comment:
A more descriptive name would be helpful since not all users will know what silent errors mean. What do you think?

Reply:
I changed it, let me know what you think.

Reply:
Looks great, thank you!